packages/teams-js/src/public/secondaryBrowser.ts

```diff
@@ -13,7 +13,8 @@ import { runtime } from './runtime';
  */
 export namespace secondaryBrowser {
   /**
-   * Open a URL in the secondary browser aka in-app browser
+   * Open a URL in the secondary browser (also known as in-app browser). 
+   * On web and desktop, the user will be redirected to an external browser.
```

Comment:
Should this be written so that it uses a parallel structure for each case, rather than using a parenthetical to describe the mobile behavior and a separate sentence to describe web/desktop? e.g.,

   * Open a URL in a secondary browser
   * On mobile platforms, this is an in-app browser 
   * On web and desktop, the user will be redirected to an external browser.

(I don't have any idea if there is a preferred style here, just a thought I had while reading it).

Reply:
While I'm at it, do we have specific terminology we use to refer to these platforms that users already understand?

I imagine most users would know what "mobile" refers to

I'm not sure if people know what "desktop" means? Would we ever say "large screen" instead?

What would users think iPad counts as?

Should we just use words like iOS, Android, Windows, Mac, and web instead of web, desktop, mobile, etc.?

I have no idea about any of this, just musing and asking questions. 😅

Reply:
Great idea (first comment) on the parallel sentence structure! Makes the differences more clear.

Desktop/Web/Mobile are pretty standard terms for tech industry content. Depending on the context, we've been enumerating individual platforms where it's helpful in Teams docs. Here in TJS reference, it's probably most pragmatic to use broad categories and rely on the [compatibility matrix](https://learn.microsoft.com/en-us/microsoftteams/platform/m365-apps/teamsjs-support-m365) to be the single source of truth on hub support details.